lib/src/kratos_client.dart

```diff
@@ -909,42 +1122,59 @@ class KratosClient {
     };
   }
 
-  Future<UserProfile> getUserProfile() async {
-    final kratosToken = await _credentialsStorage.read();
+  Future<SessionResult> getSession() async {
```

Comment:
Why the split? What is needed that the `getUserProfile` does not provide?

Reply:
Initially I wanted to use a Session to obtain a list of registration methods to know if user uses passkeys, but I found a more appropriate way later. Forgot to delete, will do